lib/resources/EnterpriseZoneWorkersKV.js

```diff
@@ -0,0 +1,126 @@
+/*
+ * Copyright (C) 2014-present Cloudflare, Inc.
+
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+'use strict';
+
+const prototypal = require('es-class');
+const auto = require('autocreate');
+
+const Resource = require('../Resource');
+const method = require('../method');
+
+/**
+ * EnterpriseZoneWorkersKV represents the accounts/:accountId/storage/kv/namespaces API endpoint.
+ *
+ * @class EnterpriseZoneWorkersKV
+ * @hideconstructor
+ * @extends Resource
+ */
+module.exports = auto(
+  prototypal({
+    extends: Resource,
+    path: 'accounts/:accountId/storage/kv/namespaces/:namespaceId',
+
+    /**
+     * browse allows for listing all the keys of a namespace
+     *
+     * @function browse
+     * @memberof EnterpriseZoneWorkersKV
+     * @instance
+     * @async
+     * @param {string} account_id - The account ID
+     * @param {string} namespace_id - The namespace ID
+     * @returns {Promise<Object>} The KV response object.
+     */
+    browse: method({
+      method: 'GET',
+      path: 'keys',
+    }),
+    /**
+     * add allows for creating a key-value pair in a namespace
+     *
+     * @function add
+     * @memberof EnterpriseZoneWorkersKV
+     * @instance
+     * @async
+     * @param {string} account_id - The account ID
+     * @param {string} namespace_id - The namespace ID
+     * @param {string} value - The value to store
```

Comment:
Shouldn't the JSDoc comment contain `@param {string} key_name` as well?

Sorry for the random drop in - I was trying to figure out how to do this via the npm package when I stumbled across this PR.

Reply:
@parris I believe the URL contains the key:

`PUT /accounts/:accountId/storage/kv/namespaces/:namespaceId/values/:id`